app/Model/User.php

```diff
@@ -0,0 +1,63 @@
+<?php
+namespace Notes\Model;
+
+class User
+{
+    public $id;
```

Comment:
data members should be protected. so only get and set can be used to read and write values of data members.